features/eolearn/features/bands_extraction.py

```diff
@@ -47,17 +47,21 @@ def map_method(self, feature):
 class NormalizedDifferenceIndexTask(MapFeatureTask):
     """ The task calculates a Normalized Difference Index (NDI) between two bands A and B as:
 
-        :math:`NDI = \\dfrac{A-B}{A+B}`
+        :math:`NDI = \\dfrac{A-B+c}{A+B+c}`,
+
+        where c is provided as the *constant* argument.
```

Comment:
Here it is very important to explain that this constant has a scientific explanation: http://www.cesbio.ups-tlse.fr/multitemp/?p=12746 . Otherwise one might thing it was added just for our convenience.
Therefore I suggest adding a bit more explanation here and the upper link.

Reply:
Done.